runtime/doc/options.txt

```diff
@@ -2831,9 +2831,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 		:s///g		  subst. one	  subst. all
 		:s///gg		  subst. all	  subst. one
 
-	DEPRECATED: Setting this option may break plugins that are not aware
-	of this option.  Also, many users get confused that adding the /g flag
-	has the opposite effect of that it normally does.
+	NOTE: Setting this option may break plugins that rely on the default
+	behavior of the 'g' flag. This will also make the 'g' flag have the
+	opposite effect of that documented in |:s_g|.
 
```

Comment:
Should `'gdefault'` also be removed from `deprecated.txt`?

Reply:
Probably; can be smooshed into the collected docs PR? @dundargoc

Reply:
Sure